src/algopy_testing/op.py

```diff
@@ -938,69 +1000,315 @@ def _get_asset_holding(
+
+        local_states = test_context._active_contract._get_local_states()
+        local_state = local_states.get(key)
+        if local_state is None:
+            raise ValueError(f"Local state with key {key.decode()} not found")
+        return local_state
+
+    def _get_key(self, b: algopy.Bytes | bytes) -> bytes:
+        import algopy
+
+        return b.value if isinstance(b, algopy.Bytes) else b
+
+    def _get_contract(
+        self,
+        b: algopy.Application | algopy.UInt64 | int,
+    ) -> algopy.Contract | algopy.ARC4Contract:
+        import algopy
+
+        from algopy_testing import get_test_context
+
+        test_context = get_test_context()
+        if not test_context:
+            raise ValueError("Test context is not initialized!")
+
+        app_id = int(b.id) if isinstance(b, algopy.Application) else int(b)
+        contract = test_context._app_id_to_contract.get(app_id)
+        if contract is None:
+            raise ValueError(f"Contract with app id {b} not found")
+        return contract
+
+    def get_bytes(
+        self, a: algopy.Account | algopy.UInt64 | int, b: algopy.Bytes | bytes, /
+    ) -> algopy.Bytes:
+        import algopy
+
+        try:
+            local_state = self._get_local_state(self._get_key(b)).get(a, 0)
+            return algopy.Bytes(local_state) if local_state else algopy.UInt64(0)  # type: ignore[return-value]
+        except (AttributeError, KeyError):
+            return algopy.UInt64(0)  # type: ignore[return-value]
+
+    def get_uint64(
+        self, a: algopy.Account | algopy.UInt64 | int, b: algopy.Bytes | bytes, /
+    ) -> algopy.UInt64:
+        import algopy
+
+        try:
+            local_state = self._get_local_state(self._get_key(b))
+            return algopy.UInt64(local_state.get(a))
+        except (AttributeError, KeyError):
+            return algopy.UInt64(0)
+
+    def get_ex_bytes(
+        self,
+        a: algopy.Account | algopy.UInt64 | int,
+        b: algopy.Application | algopy.UInt64 | int,
+        c: algopy.Bytes | bytes,
+        /,
+    ) -> tuple[algopy.Bytes, bool]:
+        import algopy
+
+        contract = self._get_contract(b)
+        local_states = contract._get_local_states()
+        local_state = local_states.get(self._get_key(c))
+        return (
+            (algopy.Bytes(local_state.get(a, b"")), True)
+            if local_state
+            else (algopy.Bytes(b""), False)
+        )
+
+    def get_ex_uint64(
+        self,
+        a: algopy.Account | algopy.UInt64 | int,
+        b: algopy.Application | algopy.UInt64 | int,
+        c: algopy.Bytes | bytes,
+        /,
+    ) -> tuple[algopy.UInt64, bool]:
+        import algopy
+
+        contract = self._get_contract(b)
+        local_states = contract._get_local_states()
+        local_state = local_states.get(self._get_key(c))
+        return (
+            (algopy.UInt64(local_state.get(a, 0)), True)
```

Comment:
Similar to `get_ex_bytes` should only return True if both account and key have a defined value